src/signals/settings/users/Detail/__tests__/Detail.test.js

```diff
@@ -469,35 +477,33 @@ describe('signals/settings/users/containers/Detail', () => {
   it('should direct to the overview page when cancel button is clicked and form data is NOT pristine', async () => {
     jest.spyOn(reactRouterDom, 'useLocation').mockImplementation(() => ({}))
 
-    global.window.confirm = jest.fn()
-
     const { findByTestId, getByTestId } = render(withAppContext(<UserDetail />))
 
     await findByTestId('user-detail-form-container')
 
     expect(navigateMock).not.toHaveBeenCalled()
 
-    act(() => {
+    await act(async () => {
       fireEvent.change(
         getByTestId('detail-user-form').querySelector('#last_name'),
         { target: { value: 'Foo Bar Baz' } }
       )
     })
 
-    act(() => {
+    await act(async () => {
```

Comment:
not needed

Reply:
removed the unnecessary awaits ...async in this file, there was one that was still necessary